src/main/java/com/vmware/vipclient/i18n/VIPCfg.java

```diff
@@ -79,34 +75,22 @@ public void setSubInstance(boolean subInstance) {
 
     private boolean isSubInstance = false;
 
-    private VIPCfg() {
+    public VIPCfg() {
```

Comment:
Why public this method? Is it necessary?

Reply:
Added the constructor for backwards compatibility, but marked it as deprecated